string port
uint16 min_pwm0
uint16 min_pwm1
uint16 max_pwm0
uint16 max_pwm1
---
bool success
